src/common/UserMessage/UserMessage.tsx

```diff
@@ -0,0 +1,13 @@
+import { TypeUserMessage } from '../../types/types';
+
+import styles from './UserMessage.module.css';
+
+const UserMessage = ({ text = '', ...props }: TypeUserMessage): JSX.Element => {
+	return (
+		<p className={styles[props.messageType]}>
```

Comment:
for the functional components please use React.FC to describe a type of the component `const Component: React.FC<PropsTypes> = (props) ={}`

Reply:
Done